loopd/start.go

```diff
@@ -27,7 +27,16 @@ var (
 	swapsLock        sync.Mutex
 )
 
-func Start() error {
+// RPCConfig holds optional options that can be used to make the loop daemon
+// communicate on custom connections.
+type RPCConfig struct {
+	// RPCListener is an optional listener that if set will override the
+	// daemon's gRPC settings, and make the gRPC server listen on this
+	// listener.
```

Comment:
Maybe mention here as well that using the custom listener will disable REST.

Reply:
Done: https://github.com/lightninglabs/loop/pull/115/commits/71be49b300985fa6dd6906b687d6abd2d9f0ffac